app/src/main/java/com/x8bit/bitwarden/ui/tools/feature/generator/GeneratorScreen.kt

```diff
@@ -596,6 +596,20 @@ private fun PasswordLengthSliderItem(
     val density = LocalDensity.current
     val sliderRange = minValue.toFloat()..maxValue.toFloat()
 
+    val lengthLabel: @Composable () -> Unit = remember {
+        {
+            Text(
+                text = stringResource(id = R.string.length),
+                modifier = Modifier
+                    .onGloballyPositioned { layoutCoordinates ->
+                        if (labelTextWidth == Dp.Unspecified) {
```

Comment:
This was getting updated each time the `OutLinedTextField` got recomposed which is why it get shrinking when the display sizes were altered. Moved it to it's own `remember` function as I was seeing a crash when going back and forth between the settings app to change the sizes. Was seeing this crash off of the main branch as well so not related to this `if` statement.